Components/Note.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import { View, StyleSheet, TextInput, Button, Text, TouchableOpacity} from 'react-native'
+import { Header, CheckBox,  } from 'react-native-elements';
+import InputNote from 'ToDoProject/Components/InputNote'
+
+export default class Note extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            check: false,
+            color: "red"
+        }
+    }
+    checkBoxTest() {
+        if(this.state.check)
+        {
+            this.setState({color:"red"})
```

Comment:
will be more readable write like this:

let color = this.state.check ? 'red' : 'green';
this.setState({color});